### ISSUE TITLE
(libreoffice-fresh) Outdated

### ISSUE THREAD

**Issue by matthewkogan** [chocolatey-community/chocolatey-packages#2359]
### Checklist

- [X] I have checked the moderation queue that no new version has been submitted (only visible when logged in), or the submitted package version has failed the automated checks.

Please navigate to the following [link](https://community.chocolatey.org/packages?q=&moderatorQueue=true&moderationStatus=all-statuses&prerelease=false&sortOrder=package-download-count) to view the moderation queue.

- [X] I have verified that the new version is a stable release.
- [X] I have looked to see if there are any Bug reports that prevents a new version being submitted and that no existing Outdated Reports have been created.
- [X] I have verified the new version has a Windows binary (EXE or MSI installer, Zip or other archive, etc).

### New Software Version

7.6.3

### Download location

https://www.libreoffice.org/download/download-libreoffice/

### Package Page

https://community.chocolatey.org/packages/libreoffice-fresh

**Comment by brogers5** [comment]
As stated in the Notes section of the package's description:

> This package relies on the update service provided by The Document Foundation to determine new updates. As their policy is to distribute to their website immediately and to the update service after 1 to 2 weeks, there will be a delay between a new software update and a new version release of the package.

The update service is still serving the latest available package version (v7.6.2) for the Fresh Branch as of now.

According to [the release notes](https://www.libreoffice.org/download/release-notes/), v7.6.3 was released on the website only 5 days ago. We may need to wait out its publication on the update service before the update script picks up on the new version.

**Comment by pauby** [comment]
Given the above comment, I'm going to go ahead and close this as there is nothing for us to do at the moment. We can always reopen it later.

**Comment by matthewkogan** [comment]
The update service now offers 7.6.4, and https://github.com/chocolatey-community/chocolatey-packages/commit/17d774e shows (I think) that it has been detected, but LibreOffice Fresh is still offering 7.6.2.

**Comment by pauby** [comment]
The version history indicates that 7.6.3 and 7.6.4 have failed verification and are waiting on the maintainer to fix.

**Comment by brogers5** [comment]
According to the gists, both versions had only timed out during the installation process. They installed without issues in my local Chocolatey Testing Environment instance. I don't see anything that needs to be fixed here.

I'd recommend either re-running the verification process or requesting a one-time exemption for them.

**Comment by matthewkogan** [comment]
Will this happen automatically tomorrow or does it require manual intervention?

**Comment by brogers5** [comment]
> Will this happen automatically tomorrow or does it require manual intervention?

It does require manual intervention, although it'd be up to either a Maintainer for this repository or a Moderator/Admin for the Community Repository to make it happen. Don't think there's much us community contributors can directly do apart from raising the issue.

**Comment by matthewkogan** [comment]
OK, can we re-open this issue then, please?

**Comment by matthewkogan** [comment]
Has somebody notified the maintainers/moderators/admins that this needs doing?

**Comment by matthewkogan** [comment]
@brogers5 @pauby Could somebody nudge a maintainer/moderator/admin to fix this, please?

**Comment by brogers5** [comment]
@matthewkogan This repository is probably the best place for getting a maintainer request in the queue. I only mentioned the Community Repository moderators/admins in particular since there is some overlap between the two, and their involvement may become necessary if the package sits long enough to be rejected. If this isn't getting attention, I'd consider opening a new issue since Closed issues may not necessarily be monitored for new comments.

Having said that, it's worth noting this is a volunteer effort and we're approaching the holiday season. You may need to be a little more patient than usual.

**Comment by matthewkogan** [comment]
Thank you. I just want to know that it's not forgotten - it's been over a week since the last comment. Could we not just re-open this issue, please?

**Comment by matthewkogan** [comment]
@brogers5 @pauby please could we re-open this issue?

**Comment by brogers5** [comment]
@pauby, as previously mentioned, I don't think there's anything that requires fixing on the package's end. I could not reproduce a timeout failure with either version. Could we rerun verification for both of them?

**Comment by pauby** [comment]
I've done that now for both versions but I suspect it will not make any difference. Hopefully I'm wrong.

**Comment by matthewkogan** [comment]
Looks like it failed again. https://gist.github.com/choco-bot/0a3ff9e5ce848538863e0ee223fd7215
Just a thought, and it could be completely irrelevant, but I see that it's using "Windows Server 2012 R2 x64" which is now end of life, and that is something that has changed since the last successful LibreOffice Fresh package validation in September.
Otherwise, is there a way for us to get hold of the libreoffice-fresh.7.6.4.MsiInstall.log file that presumably has the details of what has gone wrong?

**Comment by pauby** [comment]
> but I see that it's using "Windows Server 2012 R2 x64" 

That's a typo. The verification server is Windows Server 2019. 

There is an issue for this typo but I can't seem to find it. Tagging @Windos as he spoke about it recently.

**Comment by brogers5** [comment]
Both versions timed out during the install once again.

I retested both of them in my local testing environment instance, and did not see anything come up that would stall an unattended installation. The installation completed successfully in all cases without timing out or further input.

Having said that, the installation process does take a few minutes, so I'm inclined to believe it's just taking too long to complete in the Package Verifier environment.

Is a one-time exemption from Verifier testing an option?

**Comment by Windos** [comment]
> That's a typo. The verification server is Windows Server 2019.
> 
> There is an issue for this typo but I can't seem to find it. Tagging @Windos as he spoke about it recently.
@pauby 

This had been flagged to become public a while back, but I think one of the labels was blocking the sync from happening. The sync has now happened and the public issue is here:

https://github.com/chocolatey/home/issues/292

> I retested both of them in my local testing environment instance
@brogers5 

Out of interest, are you using your own testing environment or _the_ [testing environment](https://github.com/chocolatey-community/chocolatey-test-environment)?

**Comment by brogers5** [comment]
@Windos Just to confirm, I am using the official testing environment.

**Comment by Windos** [comment]
Evidently third time's a charm. I cannot say for certain what was causing the testing failure for two versions of this package, but they have both passed Package Verifier now.

Now that this package is no longer out of date I'll go ahead and close this issue, thanks for the report!

**Comment by matthewkogan** [comment]
@Windos Thank you.
